messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/CommonAttributes.java

```diff
@@ -177,15 +177,33 @@
             .setXmlName("param")
             .build();
 
-    SimpleAttributeDefinition JGROUPS_STACK = create("jgroups-stack", ModelType.STRING)
+    SimpleAttributeDefinition JGROUPS_CHANNEL_FACTORY = create("jgroups-stack", ModelType.STRING)
```

Comment:
it should be annotated with `@Deprecated`

Reply:
Will do.